components/webui/imports/ui/SearchView/SearchView.jsx

```diff
@@ -109,33 +117,43 @@ const SearchView = () => {
             findOptions["sort"] = sort;
         }
 
+        if ((!serverState) || !(serverState.jobID)) {
```

Comment:
```suggestion
        if (null === serverState || undefined === serverState.jobID) {
```
I am not sure if it is possible for `serverState.jobID` to be `null` as well